mage_integrations/mage_integrations/destinations/bigquery/__init__.py

```diff
@@ -114,7 +119,7 @@ def build_create_table_commands(
             create_table_command = f'''
 {create_table_command}
 PARTITION BY
-  DATE(`{clean_column_name(partition_col)}`)
+  DATE(`{clean_column_name(partition_col, self.use_lowercase)}`)
```

Comment:
I would also suggest define the clean_column_name method in the base sql destination to reduce the duplicate code.
```
def self. clean_column_name(col):
    return clean_column_name(col, lower_case=self. use_lowercase)
```

Reply:
Modified in 6b629d234dfc02cd6157d07a31dd2e14d469f4c5

MsSQL and OracleDB have their on `clean_column_name` methods in their respective utils.py, so they were not modified